docs/reference/ccr/overview.asciidoc

```diff
@@ -29,6 +29,41 @@ Replication can be configured in two ways:
 
 NOTE: You must also <<ccr-requirements,configure the leader index>>.
 
```

Comment:
I think it would be helpful to point out that all of those tasks can be done via Kibana too and add a link to `{kibana-ref}/working-remote-clusters.html#managing-cross-cluster-replication[Managing {ccr}]`

Reply:
I have added an additional note. Let me know if that looks good.

Reply:
I added a new commit that mentions Kibana directly in the bullet points, since I couldn't add a suggestion on that section.